docs/getting-started/linux-file-monitoring.asciidoc

```diff
@@ -0,0 +1,78 @@
+
+CAUTION: Ignoring file systems can create gaps in your security coverage. Use additional security layers for any file systems ignored by {endpoint-cloud-sec}.
+
+To monitor or ignore additional file systems, configure the following advanced settings related to *fanotify*, a Linux feature that monitors file system events. Go to *Manage* -> *Policies*, click a policy's name, then scroll down and select *Show advanced settings*.
+
+NOTE: Even when configured to monitor all file systems (`ignore_unknown_filesystems` is `false`), {endpoint-cloud-sec} will still ignore specific file systems that Elastic has internally identified as incompatible. The following settings apply to any _other_ file systems.
+
+[[ignore-unknown-filesystems]]
+`linux.advanced.fanotify.ignore_unknown_filesystems`:: Determines whether to ignore unrecognized file systems. Enter one of the following:
++
+--
+* `true`: (Default) Monitor only Elastic-tested file systems, and ignore all others. You can still monitor or ignore specific file systems with `monitored_filesystems` and `ignored_filesystems`, respectively.
+
+* `false`: Monitor all file systems. You can still ignore specific file systems with `ignored_filesystems`.
+--
++
+NOTE: If you've upgraded from 8.3 or earlier, this value will be `false` for backwards compatibility. If you don't need to monitor additional file systems, it's recommended to change `ignore_unknown_filesystems` to `true` after upgrading.
+
+[[monitored-filesystems]]
+`linux.advanced.fanotify.monitored_filesystems`:: Specifies additional file systems to monitor. Enter a comma-separated list of <<find-file-system-names,file system names>> as they appear in `/proc/filesystems` (for example: `jfs,ufs,ramfs`). 
++
+NOTE: It's recommended to avoid monitoring network-backed file systems.
++
+This setting isn't recognized if `ignore_unknown_filesystems` is `false`, since that would mean you're already monitoring _all_ file systems. 
++
+Entries in this setting are overridden by entries in `ignored_filesystems`.
+
+[[ignored-filesystems]]
+`linux.advanced.fanotify.ignored_filesystems`:: Specifies additional file systems to ignore. Enter a comma-separated list of <<find-file-system-names,file system names>> as they appear in `/proc/filesystems` (for example: `ext4,tmpfs`).
++
+Entries in this setting override entries in `monitored_filesystems`.
+
+[[find-file-system-names]]
+== Find file system names
+
+This section provides a few ways to determine the file system names needed for `linux.advanced.fanotify.monitored_filesystems` and `linux.advanced.fanotify.ignored_filesystems`.
+
+Assuming a typical setup, {filebeat} will be installed alongside {elastic-endpoint} and will automatically ship {elastic-endpoint} logs to {es}. {elastic-endpoint} will generate a log message about the file that was scanned when an event occurred.
```

Comment:
For my own edification, are you referring to the typical Elastic agent setup here? I'm reading this sentence as:
Assuming a typical setup, when you install Elastic Agent, Filebeat will be installed alongside Elastic Endpoint and will automatically ship Elastic Endpoint logs to Elasticsearch.

Reply:
@nastasha-solomon That's correct. Do you think this should specify installing Elastic Agent for clarity? I don't want to juggle too many proper nouns here but can definitely revise if there's a better way to explain this.

Reply:
Yeah, I think so. What you're describing sounds complex. It might be good to explicitly state that all of this activity is happening without additional user input/efforts (i.e., the Agent does it all). It could also reduce the likelihood that the sentence is misinterpreted. Someone who's unfamiliar with the Agent might interpret the sentence as "assuming you setup _Filebeat_ in the typical way, the Beat will be installed alongside Elastic Endpoint...".